python/ccxt/base/exchange.py

```diff
@@ -1093,31 +1102,12 @@ def load_accounts(self, reload=False, params={}):
         self.accountsById = self.index_by(self.accounts, 'id')
         return self.accounts
 
-    def populate_fees(self):
-        if not (hasattr(self, 'markets') or hasattr(self, 'currencies')):
-            return
-
-        for currency, data in self.currencies.items():  # try load withdrawal fees from currencies
-            if 'fee' in data and data['fee'] is not None:
-                self.fees['funding']['withdraw'][currency] = data['fee']
-                self.fees['funding']['fee_loaded'] = True
-
-        # find a way to populate trading fees from markets
-
-    def load_fees(self):
+    def load_fees(self, reload=False):
```

Comment:
We may want to also mirror this to the async version – unfortunately, have to do the duplication because of that. There's a group of base methods (like `loadMarkets`, `loadFees`, etc) in the async folder, which are good candidates for being the first transpiled base methods, because their logic is mostly the same in all languages and with both the sync and the async environments.

Reply:
will fix the rest in the morning